### PR TITLE
feat: externalizing config for Mongo

### PR DESCRIPTION
**Pull request by weliasz**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -12,14 +12,8 @@ snmp:
     # mibs_path: "/Users/yling/snmp-project/splunk-connect-for-snmp-mib-server/mibs"
 mongo:
   oid:
-    # host: "localhost"
-    host: "mongo"
-    port: 27017
     database: "snmp_traps"
```
Comment (lingy1028) [inline config.yaml:15]: I also read the db's name and collection's name from yaml. Can we make them as the Env variable, too?

Comment (weliasz) [reply config.yaml:15]: I don't think so as this is not something that Kubernetes manages.

File: splunk_connect_for_snmp_mib_server/mongo.py [hunk 2]
```diff
@@ -9,7 +9,7 @@ def __init__(self, mongo_config):
         """
         Create a collection in mongodb to store mib files
         """
-        self._client = MongoClient(host=mongo_config['host'], port=mongo_config['port'])
+        self._client = MongoClient(os.environ['MONGO_SERVICE_SERVICE_HOST'], os.environ['MONGO_SERVICE_PORT'])
         self._mibs = self._client[mongo_config['database']][mongo_config['collection']]
```
Comment (lingy1028) [inline splunk_connect_for_snmp_mib_server/mongo.py:13]: Here is the place where I create the db and collection.

File: splunk_connect_for_snmp_mib_server/mongo.py [hunk 3]
```diff
@@ -62,7 +62,7 @@ def clear(self):
 
 class OidsRepository:
     def __init__(self, mongo_config):
-        self._client = MongoClient(host=mongo_config['host'], port=mongo_config['port'])
+        self._client = MongoClient(os.environ['MONGO_SERVICE_SERVICE_HOST'], os.environ['MONGO_SERVICE_PORT'])
         self._oids = self._client[mongo_config['database']][mongo_config['collection']]
```
Comment (lingy1028) [inline splunk_connect_for_snmp_mib_server/mongo.py:66]: Here is the place where I create the db and collection.

File: config.yaml (hunk 1 above)
Comment (lingy1028) [reply config.yaml:15]: Line 7 is also needed to be referred to mib-server URL. e.g localhost needs to be replaced with the host name of mib-server.

**Review by lingy1028 — APPROVED** [review]

**Pull request merged**